lib/python/qmk/questions.py

```diff
@@ -76,17 +82,21 @@ def question(prompt, *args, default=None, confirm=False, answer_type=str, **kwar
         prompt = '%s [%s] ' % (prompt, default)
```

Comment:
The output this way feels more natural to me.

```suggestion
        prompt = '%s [%s]: ' % (prompt, default)
    else:
        prompt = '%s: ' % prompt
```

Reply:
I'm leaving control of the prompt up to the programmer. They might want to use `> ` or  `- ` (or some unicode character I don't know about,) and I decided the complexity of adding another argument to control that part of the prompt was not worth the benefit. 

I may change my mind later if we have to spend a lot of time correcting this in submissions. :)